docs/installation.md

```diff
@@ -7,7 +7,7 @@
 ## Quickstart
 
 ```bash
-$ pip3 install opsdroid
+$ pip3 install opsdroid[common]
```

Comment:
Should the quickstart install common? Or just core?

Reply:
I think since its for getting the users set up and try out opsdroid quickly, common would be a better choice

Reply:
But common only gives Slack, Matrix and sqlite right? And the quickstart docs do not mention those at all.

I expect many users will be unfamiliar with extra packages specified this way. Opsdroid it aimed at novice users. I just feel like this is unnecessary complexity at this point.

Perhaps this is a sign that we should rethink the quickstart. What do we actually want to show people here?